src/thor/triplegbuilder.cc

```diff
@@ -1258,21 +1484,16 @@ void TripLegBuilder::Build(
 
   // Iterate through path
   bool is_first_edge = true;
-  uint32_t block_id = 0;
   uint32_t prior_opp_local_index = -1;
   std::vector<PointLL> trip_shape;
-  std::string arrival_time;
-  bool assumed_schedule = false;
   uint64_t osmchangeset = 0;
   size_t edge_index = 0;
   const DirectedEdge* prev_de = nullptr;
   const GraphTile* graphtile = nullptr;
-  // TODO: this is temp until we use transit stop type from transitland
-  TransitPlatformInfo_Type prev_transit_node_type = TransitPlatformInfo_Type_kStop;
+  MultimodalBuilder multimodal_builder(origin, time_info);
```

Comment:
all of the variables removed above are now member variables of the struct, intialized in the same way as they are here